report.php

```diff
@@ -112,7 +105,9 @@
     }
     $table->head[] = get_string('info', 'logstore_xapi');
     $table->head[] = get_string('datetimegmt', 'logstore_xapi');
+    $table->head[] = '';
     $table->id = "report";
+    $replayevent = get_string('replayevent', 'logstore_xapi');
```

Comment:
Is this variable used?

Reply:
Nope, good catch. It was moved to the other place. I will remove it.